tests/kafkatest/services/security/security_config.py

```diff
@@ -259,6 +267,9 @@ def setup_node(self, node):
         if self.has_sasl:
             self.setup_sasl(node)
 
+        if java_version(node) <= 9 and self.properties['tls.version'] == 'TLSv1.3':
```

Comment:
For consistency, shall we use `11` here?

Reply:
OK. Fixed.